Sources/OAuthSwiftCredential.swift

```diff
@@ -140,34 +150,81 @@ open class OAuthSwiftCredential: NSObject, NSCoding {
-        self.oauthTokenExpiresAt = (decoder.decodeObject(forKey: CodingKeys.oauthTokenExpiresAt) as? Date)
-        self.version = Version(decoder.decodeInt32(forKey: CodingKeys.version))
+        self.consumerKey = (decoder.decodeObject(forKey: NSCodingKeys.consumerKey) as? String) ?? String()
+        self.consumerSecret = (decoder.decodeObject(forKey: NSCodingKeys.consumerSecret) as? String) ?? String()
+        self.oauthToken = (decoder.decodeObject(forKey: NSCodingKeys.oauthToken) as? String) ?? String()
+        self.oauthRefreshToken = (decoder.decodeObject(forKey: NSCodingKeys.oauthRefreshToken) as? String) ?? String()
+        self.oauthTokenSecret = (decoder.decodeObject(forKey: NSCodingKeys.oauthTokenSecret) as? String) ?? String()
+        self.oauthVerifier = (decoder.decodeObject(forKey: NSCodingKeys.oauthVerifier) as? String) ?? String()
+        self.oauthTokenExpiresAt = (decoder.decodeObject(forKey: NSCodingKeys.oauthTokenExpiresAt) as? Date)
+        self.version = Version(decoder.decodeInt32(forKey: NSCodingKeys.version))
         if case .oauth1 = version {
-            self.signatureMethod = SignatureMethod(rawValue: decoder.decodeObject(forKey: CodingKeys.signatureMethod) as? String ?? "HMAC_SHA1") ?? .HMAC_SHA1
+            self.signatureMethod = SignatureMethod(rawValue: decoder.decodeObject(forKey: NSCodingKeys.signatureMethod) as? String ?? "HMAC_SHA1") ?? .HMAC_SHA1
         }
     }
 
     open func encode(with coder: NSCoder) {
-        coder.encode(self.consumerKey, forKey: CodingKeys.consumerKey)
-        coder.encode(self.consumerSecret, forKey: CodingKeys.consumerSecret)
-        coder.encode(self.oauthToken, forKey: CodingKeys.oauthToken)
-        coder.encode(self.oauthRefreshToken, forKey: CodingKeys.oauthRefreshToken)
-        coder.encode(self.oauthTokenSecret, forKey: CodingKeys.oauthTokenSecret)
-        coder.encode(self.oauthVerifier, forKey: CodingKeys.oauthVerifier)
-        coder.encode(self.oauthTokenExpiresAt, forKey: CodingKeys.oauthTokenExpiresAt)
-        coder.encode(self.version.toInt32, forKey: CodingKeys.version)
+        coder.encode(self.consumerKey, forKey: NSCodingKeys.consumerKey)
+        coder.encode(self.consumerSecret, forKey: NSCodingKeys.consumerSecret)
+        coder.encode(self.oauthToken, forKey: NSCodingKeys.oauthToken)
+        coder.encode(self.oauthRefreshToken, forKey: NSCodingKeys.oauthRefreshToken)
+        coder.encode(self.oauthTokenSecret, forKey: NSCodingKeys.oauthTokenSecret)
+        coder.encode(self.oauthVerifier, forKey: NSCodingKeys.oauthVerifier)
+        coder.encode(self.oauthTokenExpiresAt, forKey: NSCodingKeys.oauthTokenExpiresAt)
+        coder.encode(self.version.toInt32, forKey: NSCodingKeys.version)
         if case .oauth1 = version {
-            coder.encode(self.signatureMethod.rawValue, forKey: CodingKeys.version)
+            coder.encode(self.signatureMethod.rawValue, forKey: NSCodingKeys.version)
```

Comment:
If my assumption is correct in PR #438, the key here will need to be updated to `NSCodingKeys.signatureMethod`